tinytuya/BulbDevice.py

```diff
@@ -136,36 +121,33 @@ def _rgb_to_hexvalue(r, g, b, bulb="A"):
 
         # Bulb Type A
         if bulb == "A":
-            # h:0-360,s:0-255,v:0-255|hsv|
             hexvalue = ""
-            for value in rgb:
-                temp = str(hex(int(value))).replace("0x", "")
-                if len(temp) == 1:
-                    temp = "0" + temp
-                hexvalue = hexvalue + temp
-
-            hsvarray = [int(hsv[0] * 360), int(hsv[1] * 255), int(hsv[2] * 255)]
-            hexvalue_hsv = ""
-            for value in hsvarray:
-                temp = str(hex(int(value))).replace("0x", "")
-                if len(temp) == 1:
-                    temp = "0" + temp
-                hexvalue_hsv = hexvalue_hsv + temp
-            if len(hexvalue_hsv) == 7:
-                hexvalue = hexvalue + "0" + hexvalue_hsv
-            else:
-                hexvalue = hexvalue + "00" + hexvalue_hsv

```

Comment:
I love that we are addressing this. This is original pytuya code and more complicated than it needs to be. Thank you.